pages/blog/[category]/[slug]/index.tsx

```diff
@@ -0,0 +1,85 @@
+import type { Metadata, NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import Post, { Article } from '../../../../components/blog/Post';
+import Footer from '../../../../components/Footer';
+import { fetchAPI } from '../../../../utils/blog/fetch-api';
+import Navbar from '../../../../views/Navbar';
+
+async function getPostBySlug(slug: string) {
+	const path = `/articles`;
+	const urlParamsObject = {
+		filters: { slug },
+		populate: {
+			cover: { fields: ['url'] },
+			authorsBio: { populate: '*' },
+			category: { fields: ['name'] },
+			blocks: { populate: '*' },
+		},
+	};
+	const response = await fetchAPI(path, urlParamsObject);
```

Comment:
No error handling?
When you called the `getPostBySlug` function, there also I can't see any error handling.